packages/react-ui/__tests__/widgets/legend/LegendProportion.test.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import LegendProportion from '../../../src/widgets/legend/LegendProportion';
+
+const LEGEND_WITH_STATS = {
+  stats: {
+    min: 0,
+    max: 200
+  }
+};
+
+const LEGEND_WITH_LABELS = {
+  labels: ['0', '200']
```

Comment:
This component API is very misleading, with these string labels that need to be actual numbers…